GliaWidgets/Sources/View/Chat/ChatView.swift

```diff
@@ -501,11 +501,11 @@ extension ChatView {
             let textView = UITextView()
             textView.text = "Quick Reply: \(button.content)"
             return .gvaQuickReply(textView)
-        case let .gvaGallary(_, gallary):
+        case let .gvaGallery(_, gallery):
             // Temporary, since UI hasn't been implemented
 
             let textView = UITextView()
-            textView.text = "Gallary: \(gallary.type.rawValue)"
+            textView.text = "Gallary: \(gallery.type.rawValue)"
```

Comment:
Here in the text Gallery also

Reply:
[b8c9397](https://github.com/salemove/ios-sdk-widgets/pull/667/commits/b8c9397c55726bec088bd3184cdad07c0b928d27)